addon/services/tour.js

```diff
@@ -350,20 +350,29 @@ export default Service.extend(Evented, {
   },
 
   /**
-   * Increases the z-index of the element, to pop it out above the overlay and highlight it
+   * Modulates the styles of the passed step's target element, based on the step's options and
+   * the tour's `modal` option, to visually emphasize the element
    *
    * @param step The step object that attaches to the element
    * @private
    */
   popoutElement(step) {
-    $('.shepherd-modal').removeClass('shepherd-modal');
     const currentElement = this.getElementForStep(step);
 
     if (currentElement) {
-      $(currentElement).addClass('shepherd-modal');
       if (step.options.highlightClass) {
         $(currentElement).addClass(step.options.highlightClass);
       }
+
+      if (this.get('modal')) {
+        currentElement.style.pointerEvents = 'none';
+        $('.shepherd-modal').removeClass('shepherd-modal');
+        $(currentElement).addClass('shepherd-modal');
+      }
+
+      if (step.options.copyStyles) {
```

Comment:
This should also be moved inside the `if (this.get('modal')) {` block. We will definitely not want to copy styles if modal is not enabled.